protokt-runtime/src/commonMain/kotlin/com/toasttab/protokt/rt/KtMessage.kt

```diff
@@ -24,4 +25,6 @@ expect interface KtMessage {
     fun serialize(serializer: KtMessageSerializer)
 
     fun serialize(): ByteArray
+
+    fun serializeToBytes(): Bytes
```

Comment:
This felt appropriate to have so you can have a no-copy way to build Any instances. Unless there's another clever way of doing it, but I don't see it.

Reply:
Could be `Bytes.from(message: KtMessage)` and directly invoke the constructor to avoid polluting this interface.